### PR TITLE
Added Adaptive Icons

### LINKED ISSUE #16: App Icon
Update the App icon according to the Android 8 Adaptive icons 
https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive
[Logo SVG asset](https://github.com/home-assistant/home-assistant-android/files/3821857/logo.svg.zip)

### PR DESCRIPTION
**Pull request by BCNelson**
Fixes #16

### CODE REVIEW

**Comment by balloob** [comment]
That's awesome. Great first contribution! Welcome to the Home Assistant Community 👋 

Can you post a screenshot?

**Comment by BCNelson** [comment]
Screenshot_1573246649

**Comment by CedrickFlocon** [comment]
Hey, Thanks for your contribution !
I have some feedback : 
- [ ] I think we do not need this file `app/src/main/ic_launcher-web.png` it's only for the [playstore](https://support.google.com/googleplay/android-developer/answer/1078870?hl=en)
- [ ] This `app/src/main/res/drawable/ic_logo.xml` is not used and duplicate of `app/src/main/res/drawable/ic_launcher_foreground.xml`.
- [ ] `app/src/main/res/.../ic_launcher_round.xml` is not referenced in the [masnifest](https://developer.android.com/about/versions/nougat/android-7.1#circular-icons)
 - [ ] Could you use the `colorPrimary` instead of `ic_launcher_background` (and remove the file `app/src/main/res/values/ic_launcher_background.xml`
- [ ] We can also drop the compatibility with low resolution `ldpi` `mdpi`

I am open to discussion (and not an expert of android asset)

File: app/src/main/res/values/ic_launcher_background.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="ic_launcher_background">#03A9F4</color>
```
Comment (CedrickFlocon) [inline app/src/main/res/values/ic_launcher_background.xml]: You forgot to remove this file

File: app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@color/ic_launcher_background"/>
```
Comment (TheZoker) [inline app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml]: I guess the color needs to be fixed here as well:
```xml
<background android:drawable="@color/colorPrimary"/>
```

**Pull request merged**